### PR TITLE
Update Vision Sensor Topical Tutorial

### LINKED ISSUE #70: update vision sensor topical tutorial
With the release of 3.1.4, users now have the ability to set color signatures and codes programmatically. The topical tutorial should be updated to take these features into account, particularly the color code functionality.

Also we should clarify that the vision utility is the standalone one (which is installed on Windows by the PROS installer), and not the one packaged with VCS (which doesn't seem to save data to the camera).

### PR DESCRIPTION
**Pull request by kunwarsahni01**
Summary:
- Shows how to create signatures programmatically. 
- Shows how to create color codes. 

Closed #70

### CODE REVIEW

File: v5/tutorials/topical/vision.rst [hunk 1]
```diff
@@ -9,7 +9,74 @@ Setting Signatures
 ==================
 
 The first step to using the vision sensor is setting color signatures for the sensor
-to recognize as objects. This is done through the V5 Vision Utility program.
+to recognize as objects. This is done through the V5 Vision Utility program or programmticly.
```
Comment (HotelCalifornia) [inline v5/tutorials/topical/vision.rst:12]:
```suggestion
to recognize as objects. This is done through the V5 Vision Utility program or programmatically.

```

**Review by HotelCalifornia — CHANGES_REQUESTED** [review]
minor typo but otherwise it looks good

File: v5/tutorials/topical/vision.rst [hunk 2]
```diff
@@ -9,7 +9,75 @@ Setting Signatures
...
+      .. code-block:: c
+         :caption: opcontrol.c
+         :linenos:
+
+         #define VISION_PORT 1
+
+         void opcontrol() {
+           // values acquired from the vision utility
+           vision_signature_s_t RED_SIG =
+           vision_signature_from_utility(EXAMPLE_SIG, 8973, 11143, 10058, -2119, -1053, -1586, 5.4, 0);
```
Comment (nathan-moore) [inline v5/tutorials/topical/vision.rst:27]:
```suggestion
             vision_signature_from_utility(EXAMPLE_SIG, 8973, 11143, 10058, -2119, -1053, -1586, 5.4, 0);

```

File: v5/tutorials/topical/vision.rst [hunk 3]
```diff
@@ -9,7 +9,75 @@ Setting Signatures
...
+         :linenos:
+
+         #define VISION_PORT 1
+         #define EXAMPLE_SIG 1
+
+         void opcontrol() {
+           pros::Vision vision_sensor (VISION_PORT);
+           
+           pros::vision_signature_s_t RED_SIG =
+           pros::Vision::signature_from_utility(EXAMPLE_SIG, 8973, 11143, 10058, -2119, -1053, -1586, 5.4, 0);
```
Comment (nathan-moore) [inline v5/tutorials/topical/vision.rst:43]:
```suggestion
             pros::Vision::signature_from_utility(EXAMPLE_SIG, 8973, 11143, 10058, -2119, -1053, -1586, 5.4, 0);

```

File: v5/tutorials/topical/vision.rst (hunk 2 above)
Comment (nathan-moore) [reply v5/tutorials/topical/vision.rst:27]: Do you also want to call set_signature here?

**Review by nathan-moore — APPROVED** [review]

**Pull request merged**